OpenLDAP/openldap/_meta/smart-descriptions.json

```
@@ -0,0 +1,41 @@
+[
+  {
+    "value": "Establish a connection from {source.ip} to {destination.ip}",
```

I suggest harmonizing the message with ones in others formats.
```suggestion
    "value": "Connection from {source.ip} to {destination.ip}",
```